### ISSUE TITLE
ExcludeKeys from nested objects in queries

### ISSUE THREAD

**Issue by pixael** [parse-community/parse-server#7390]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Current Limitation

Excluding keys in nested objects.

### Feature / Enhancement Description

Exclude keys of nested objects using the syntax object.key.

### Example Use Case

```
curl -X GET \
  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
  -H "X-Parse-REST-API-Key: ${REST_API_KEY}" \
  -G \
  --data-urlencode 'excludeKeys=player.profile' \
  https://YOUR.PARSE-SERVER.HERE/parse/classes/GameScore/Ed1nuqPvcm
```

### Alternatives / Workarounds

I've not found any workaround. I'd need to esclude a heavy nested object from a query but can't at the moment.

**Comment by mtrezza** [comment]
Thanks for suggesting.

Excluding fields in embedded documents is natively supported by MongoDB and seems to be a useful addition to Parse Server. Feel free to open a PR to add this feature and ping if you need any help.

**Comment by cbaker6** [comment]
This is already fixed in #7242

**Comment by pixael** [comment]
> This is already fixed in #7242

Thanks Corey for making this change, I see that the PR is still in review... I hope it will be merged soon.

**Comment by mtrezza** [comment]
@cbaker6 Where is this described in https://github.com/parse-community/parse-server/pull/7242 or the related issue https://github.com/parse-community/parse-server/issues/7245?

**Comment by cbaker6** [comment]
> @cbaker6 Where is this described in #7242 or the related issue #7245?
> 
> According to the descriptions there they seem to focus on a different issue.

It’s mentioned at the bottom of todos and linked to a related issues:

> Fix excluding nested fields (Fix #7030)

In that issue, the last comment is from @dplewis asks me to look into it because of the PR I had open.

The fix covers the issue mentioned here in the testcases, as well as deeper nested. This issue was brought up on https://community.parseplatform.org/t/deselect-pointer-key/1592

**Comment by mtrezza** [comment]
Thanks, so this issue is a duplicate of https://github.com/parse-community/parse-server/issues/7030.

I am closing this as duplicate.

Side note, #7030 would be better fixed in a separate PR, instead of being mixed into a PR that already seems pretty convoluted and deals with a different issue. I will add a comment about this in #7030.
